pycbc/events/stat.py

```diff
@@ -132,6 +132,25 @@ def coinc(self, s0, s1, slide, step): # pylint:disable=unused-argument
         """
         return (s0**2. + s1**2.) ** 0.5
 
+    def coinc_multiifo(self, s, slide, step,
+                       ): # pylint:disable=unused-argument
+        """Calculate the coincident detection statistic.
+        Parameters
+        ----------
+        s: dictionary keyed by ifo of single detector ranking
+           statistics
+        slide: (unused in this statistic)
+        step: (unused in this statistic)
+        Returns
+        -------
+        numpy.ndarray
+            Array of coincident ranking statistic values
+        """
+        newsnr=0
+        for i in s:
+            newsnr = newsnr + s[i]**2.
+        return newsnr ** 0.5
```

Comment:
Maybe compress all of this into a single line list comprehension:
```
return (sum([s[i]**2 for i in s]))**0.5
```

I wonder if we should have a `coinc` function *and* a `coinc_multiifo` function, but possibly the design of this will depend on some of the more complicated statistics in this module. So I'm happy to merge this to allow some form of triple coincident code to run,